README.md

```diff
@@ -5,38 +5,67 @@
 
 <p align="center"><img src="assets/sc-timetable.png" style="border-radius: 1rem" width="500"/></p>
 
-## About the project
+# Compa
 
-Compa aims to be the resource-central for higher education. As a fresher, easing into the school system has lots of friction. Finding interest groups/clubs, timetable, starting conversations, learning from seniors and past students is simply difficult.
+A companion application for students to manage and access resources at their higher education institution.
+
+## About
+
+As a fresher, settling into school could be easier. You usually need to find:
+
+- Groups/Clubs you are interested in
+- Semester timetables
+- Resources from past semesters
+
+For continuing students, it's just as hard to track where documents or class notes are when conversations are strewn across multiple platforms.
+
+Being a student is hard enough, that's why Compa aims to be _resource-central_ for higher education institutions and save you from one source of stress.
 
 As continuing students, keeping up with any of these is scattered across multiple platforms. Compa aims to offer a cohesive experience for all of these.
 
-Companion, _compass_, compact and open. Compa is all of these and more.
+Your open, compact, companion and compass. That is _Compa_.
 
 ## Features
 
-- [x] Timetable: See lecture schedule for a semester and download to your calendar. ❇️
+- [x] Timetable: See the lecture schedule for a semester and import it to your calendar. ❇️
+
+- [ ] Discussions: Ask questions, share ideas and interact with other students.
 
-- [ ] Discussions: Ask questions, share ideas and comments with other students. (Coming next). Discussions will be threaded and searchable. They can also be scoped to a specific programme, course, level, etc.
+- [ ] Communities: Find and join groups that interest you.
 
-  With discussions, you can add polls, flashcards, quizzes and files/docs.
+## Supported Schools
```

Comment:
Supported schools is a misnomer. My mind interprets it as these are schools compa supports implying there are some that it doesn't.

Schools is just fine.

---

Speak of schools. On the website, we'll need to have section: Schools directory